packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -112,8 +116,16 @@ class GuildBanManager extends CachedManager {
     return this._add(data, cache);
   }
 
-  async _fetchMany(cache) {
-    const data = await this.client.rest.get(Routes.guildBans(this.guild.id));
+  async _fetchMany({ cache, limit, before, after } = {}) {
+    const options = {};
+    if (limit) options.limit = limit;
+    if (before) options.before = before;
+    if (after) options.after = after;
+
+    const data = await this.client.rest.get(Routes.guildBans(this.guild.id), {
+      query: new URLSearchParams(options),
+    });
```

Comment:
You can just reuse the passed object here.

```suggestion
  async _fetchMany(options) {
    const data = await this.client.rest.get(Routes.guildBans(this.guild.id), {
      query: new URLSearchParams(options ?? {}),
    });
```

Reply:
Performing `<GuildBanManager>.fetch({ limit: 5 })` with your suggestion will cause Discord to reject the request as the other values are being explicitly sent as `undefined`.

Reply:
It wouldn't, in the code snippet you gave you never set the other missing props as `undefined` so they don't get serialized at all. See [this playground](https://www.typescriptlang.org/play?#code/FAFwngDgpgBA8hEBLA9gOwM4wLwwN7AxEwBmKKA-AFwwYgBOSaA5gNyHEBGAhvdbQyZtgAX2DASAVzQBjZOhggodABQpEqTDQTzMASnwciM9BhQAbKADpzKZirRQA7jACqAJQAyAZSi8ZABYACrzcALYYahqmelYgKN6CLCp6eqLiSqoExKTkNADk6FD5onpAA)

Reply:
Things fail when you add an undefined key. See [this other playground](https://www.typescriptlang.org/play?#code/FAFwngDgpgBA8hEBLA9gOwM4wLwwN7AxEwBmKKA-AFwwYgBOSaA5gNyHEBGAhvdbQyZtgAX2DASAVzQBjZOhggodABQpEqTDQTzMASnwciM9BhQAbKADpzKZirRQA7jACqAJQAyAZSi8ZABYACrzcALYYahqmelYgKN6CLCp6eqLiSqoExKTkNADk6FD5ADQwPPQ00gAmUCRMUNWiekA).